aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -20,16 +20,209 @@ This article is under development and not complete. More information may be foun
 
-<!--
 ## Blazor
+
+### Full-stack web UI
+
+With the release of .NET 8, Blazor is a full-stack web UI framework for developing apps that render content at either the component or page level with:
+
+* Static server rendering to generate static HTML.
+* Interactive server rendering using the Blazor Server hosting model.
+* Interactive client rendering using the Blazor WebAssembly hosting model.
+* Automatic interactive client rendering using Blazor Server initially and then WebAssembly on subsequent visits after the Blazor bundle is downloaded and the .NET WebAssembly runtime activates. Automatic rendering usually provides the fastest app startup experience.
+
+Interactive render modes also prerender content by default.
+
+For more information, see <xref:blazor/components/render-modes?view=aspnetcore-8.0&preserve-view=true>.
+
+### New Blazor Web App template
+
+We've introduced a new Blazor project template: the *Blazor Web App* template. The new template provides a single starting point for using Blazor components to build any style of web UI. The template combines the strengths of the existing Blazor Server and Blazor WebAssembly hosting models with the new Blazor capabilities added in .NET 8: static server rendering, streaming rendering, enhanced navigation and form handling, and the ability to add interactivity using either Blazor Server or Blazor WebAssembly on a per-component basis.
+
+As part of unifying the various Blazor hosting models into a single model in .NET 8, we're also consolidating the number of Blazor project templates. We removed the Blazor Server template, and the ASP.NET Core Hosted option has been removed from the Blazor WebAssembly template. Both of these scenarios are represented by options when using the Blazor Web App template.
+
+> [!NOTE]
+> Existing Blazor Server and Blazor WebAssembly apps remain supported in .NET 8. Optionally, these apps can be  updated to use the new full-stack web UI Blazor features.
+
+For more information on the new Blazor Web App template, see the following articles:
+
+* <xref:blazor/tooling?view=aspnetcore-8.0&pivots=windows&preserve-view=true>
+* <xref:blazor/project-structure?view=aspnetcore-8.0&preserve-view=true>
+
+### Enhanced navigation and form handling
+
+Static server rendering typically performs a full page refresh whenever the user navigates to a new page or submits a form. In .NET 8, Blazor can enhance page navigation and form handling by intercepting the request and performing a fetch request instead. Blazor then handles the rendered response content by patching it into the browser DOM. Enhanced navigation and form handling avoids the need for a full page refresh and preserves more of the page state, so pages load faster and more smoothly. Enhanced navigation is enabled by default when the Blazor script (`blazor.web.js`) is loaded. Enhanced form handling can be optionally enabled for specific forms.
+
+### Support for multiple forms and antiforgery
```

Comment:
I think this section on handling traditional forms should go before the enhanced navigation and form handling section.